### PR TITLE
NXCM-3564, adding nexus-proxy profile to run UT's against http client 4 (similar to the ahc tests).  Fixes a couple small problems that prevented the tests from passing.

### PR DESCRIPTION
**Pull request by bdemers**

### CODE REVIEW

**Comment by nabcos** [comment]
+1

**Comment by peterlynch** [comment]
+1

File: nexus/nexus-proxy/pom.xml [hunk 1]
```diff
@@ -277,6 +277,32 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>apacheHttpClient4x-provider</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
```
Comment (velo) [inline nexus/nexus-proxy/pom.xml:286]: Do we really need to re-run ALL tests or should it be just a set of test?

**Comment by velo** [comment]
+1

**Pull request merged**